build.gradle

```diff
@@ -175,6 +176,8 @@ project('spring-amqp') {
 	dependencies {
 
 		compile "org.springframework:spring-core:$springVersion"
+		compile "org.springframework:spring-web:$springVersion"
+		compile "org.apache.httpcomponents:httpclient:$httpClientVersion"
```

Comment:
First of all `optional`. If I am not interested in that `RabbitManagementTemplate` I would not like to have these dependencies.
From other side: the wrong `project`. The `RabbitManagementTemplate` is really for RabbitMQ, so move these to the `spring-rabbit` project, please.